requirements.txt

```diff
@@ -24,7 +24,7 @@ constantly==15.1.0
 coverage==6.5.0
 cycler==0.11.0
 dm-tree==0.1.7
-eclipse-sumo==1.10.0
+eclipse-sumo==1.12.0
```

Comment:
No need to modify the `requirements.txt` file, as it will be automatically overwritten by the CI `ci-auto-commit-linux.yml`.